scss/display-layout.scss

```diff
@@ -1,5 +1,6 @@
 .content {
   margin: $nav-bar-height auto;
+  padding: 20px 0;
```

Comment:
you should use 20px in the sides too. In small resolutions this will help making space to the sides and preventing elements to be stuck in the borders.